### ISSUE TITLE
passing an empty environment file throws an exception

### ISSUE THREAD

**Issue by Victoremepunto** [bsquizz/ocdeployer#56]
The deploy command dies with a stack trace message and this exception: 

ValueError: File '/home/vmugicag/dev/projects/e2e-deploy/env/builder-env.yml' is empty!

if the environment file is empty. I believe it should either ignore it, or fail (more) gracefully

**Comment by bsquizz** [comment]
To clarify, this is just if the file is present but empty, right?

If it is not present at all, we handle that scenario fine correct?

**Comment by Victoremepunto** [comment]
Yeah, well ... this is passing in an existing file that is “entirely” empty. I hit this commenting out all lines in the doc.

 So it’s not technically empty but the contents after removing comments are empty
